src/kayenta/stages/kayentaStage/kayentaStage.controller.ts

```diff
@@ -371,8 +371,8 @@ export class KayentaStageController implements IComponentController {
 
   private loadProviders = async (): Promise<string[]> => {
     const providers = await AccountService.listProviders(this.$scope.application);
-    // TODO(dpeach): allow providers other than gce.
-    return this.providers = providers.filter(p => p === 'gce');
+    // TODO: Open up to all providers.
+    return this.providers = providers.filter(p => ['gce', 'aws', 'titus'].includes(p));;
```

Comment:
@danielpeach I left this as a whitelist since I assumed you were hesitant to open up to K8s, etc. If you're comfortable relaxing it to any provider I'm happy to pull it out.

Reply:
Yep, that's right. Thanks.